### ISSUE TITLE
Overflow menu is not accessible in Windows High Contrast mode on Firefox

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6889]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
The overflow menu has the following accessibility errors on Firefox and Edge
- the icon disappears
- on Firefox, there is no perceivable focus state

Firefox
![image](https://user-images.githubusercontent.com/40970507/93942446-0688bb80-fcf6-11ea-82c2-e19e21caba86.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93942454-0b4d6f80-fcf6-11ea-9c7c-1dcf11d29ac1.png)
